Removes "Special Verbs" category & reorganises admin buttons categories

peoplearestrange opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

I've re-organised all the admin buttons into 3 new categories and kept 2 of the old ones
NEW:
Administration = All "paperwork" style tools and base commands for players
Admin - Game = All investigative tools and admin tools that effect the game
Admin - Events = All tools used to make/manage/build on IC concepts
SAME:
Fun = Game Breaking Meme's we all love
Server = ACTUAL server effecting commands, ones that effect OOC and server stuff

I've also moved a few outliers to OOC (Link Discord, Notify restart) and debug (delete and check contents). As these are either used by players or used only for debugging really

I also completely removed the "Hide Most" button verb as its now simply not needed given most buttons have spread out and its less of a cluster cuss. I left the "hide all" as it seems to squash it down to the bare min nicely. This is now in the "Administration tab"

Here is a breakdown of all the movements:
https://docs.google.com/spreadsheets/d/1JAwY93VkfdcquTJqkFEu2hO5TB0jF6ydKKnTXPTSvZg/edit?usp=sharing

## Why It's Good For The Game

I realise this is going to upset admins at first, my god do we all hate change. But the organisation is much more logical and removes the god awful "Special Verbs" tab which honestly was a bastard of left over bits.

It also allows for more obviously labelled categories in the tabs, and thus reducing buttons in each tab.

## Changelog
:cl:
del: Removed Special Verbs Tab
admin: New admin button Tabs
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

peoplearestrange committed:
Changes Category

Administration

peoplearestrange committed:
Changes Category

Admin - Game

peoplearestrange committed:
Changes category 

Administration

peoplearestrange committed:
Changes Category

Admin - Event

peoplearestrange committed:
Changes Category

Admin - Event

peoplearestrange committed:
Changes Category

Administration

peoplearestrange committed:
Changes category

Admin - Event

peoplearestrange committed:
Update one_click_antag.dm

peoplearestrange committed:
Update beakerpanel.dm

peoplearestrange committed:
Changes Category

Admin - Game

peoplearestrange committed:
Changes Category

Subtle Message = Admin - Events
Headset message = Admin - Events
Modify Antagonist Reputation =  null
Global narrate = Admin - Events
Direct Narrate = Admin - Events
Local Narrate = Admin - Events
Godmode = Admin - Game
Respawn Character - Admin - Game
Add Custom AI law = Admin - Events
Rejuvenate = Debug
Create Command Report = Admin - Events
Delete - Debug
Manage Job Slots - Admin - Game
Explosion = Fun
EM Pulse = Fun
Gib = Fun
Gibself = Fun
Check contents = Debug
Change View Range = Admin - Game
Call Shuttle = Admin - Events
Cancel Shuttle = Admin - Events
Set Security Level = Admin - Events
Toggle Nuke = Admin - Events
Toggle Combo HUD = Admin - Game
Run Weather = Admin - Events
Show Tip = Administration
Player Playtime = Administration

peoplearestrange committed:
Changes Category

Server

peoplearestrange committed:
Changes Category

Server

peoplearestrange committed:
Changes category

Admin - Events

peoplearestrange committed:
Changes Category

Grant full access = Debug
Assume direct control = Admin - Game
Give direct control = Admin - Game

peoplearestrange committed:
Changes Category

Reload Admins = Administration

peoplearestrange committed:
Changes Category

Jump to area = Admin - Game
Jump to turf = Admin - Game
Jump to Mob = Admin - Game
Jump to Coordinate = Admin - Game
Jump to key = Admin - Game
Get mob = Admin - Game
Get key = Admin - Game
Send mob = Admin - Game

peoplearestrange committed:
Changes Category

admin pm = Administration

peoplearestrange committed:
Changes Category

Admin - Game

peoplearestrange committed:
Changes Category

Admin - Game

peoplearestrange committed:
Changes Category

Get server logs = Administration
Get current logs = Administration

peoplearestrange committed:
Changes Category

Show player panel = Admin - Game
Access Newscaster Network = Admin - Events
Announce = Administration
Set Admin Notice = Administration
Start now = Admin - Events
Delay pre-game = Admin - Events
Unprison = Administration
Show Traitor Panel = Admin - Game

peoplearestrange committed:
Changes category

investigate = admin - game

peoplearestrange committed:
Removes hide most & changes category

Adminverbs - Hide Most [REMOVED] (not needed after reshuffle)
Adminverbs - Hide All = Administration
Adminverbs - Show = Administration
aghost = Admin - Game
Invisimin = Admin - Game
Check Antagonists = Admin - Game
Banning Panel = Administration
Unbanning panel = Administration
Game panel = Admin - Game
Secrets = Admin - Game
Stealth mode = Administration
Drop Bomb = Fun
Drop dynEx bomb = Fun
Osay = Admin - Events
Toggle Build Mode Self = Admin - Events
Check AI Laws = Admin - Game
Deadmin = Administration
Readmin = Administration
Toggle Admin Ai Interact = Admin - Game

peoplearestrange committed:
Changes Category

Admin - Game

peoplearestrange committed:
Changes Category

Administration

peoplearestrange committed:
Changes Category

Administration

peoplearestrange committed:
Changes Category

Administration

peoplearestrange committed:
Changes Category

Administration

peoplearestrange committed:
Changes Category

OOC

peoplearestrange committed:
Changes Category

OOC

peoplearestrange committed:
Changes Category

Set player OOC color = Server
Reset player OOC color = Server
Adminnotice = Administration

peoplearestrange committed:
Changes Category

Adminwho = Administration

peoplearestrange committed:
Change category

Load Away Mission = Admin - Events

peoplearestrange committed:
Changes Category

Config/Launch Supplypod = Admin - Events

peoplearestrange committed:
Changes Category

Trigger Event = Admin - Events

peoplearestrange committed:
Changes Category

Administration

peoplearestrange committed:
Changed Category

Select Equipment = Admin - Events

optimumtact commented:
based change

optimumtact commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -296,20 +296,11 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 		/client/proc/readmin
 		)
 
-/client/proc/hide_most_verbs()//Allows you to keep some functionality while hiding some verbs
```
why did you remove this

peoplearestrange replied:
Apparently because Im over tired... Ill add it back

optimumtact replied:
tbh looks pretty useless

peoplearestrange replied:
Oh now I see, I removed too much, One extra line got caught in the delete of the "Hide most"

peoplearestrange committed:
Update admin_verbs.dm

kriskog commented:
Getting used to new positions is going to be a pain, but it needed to be done. :+1:

Dorsisdwarf commented:
if an admin is upset at the verb tornado being tamed I will destroy their very soul 👍

cacogen commented:
you need some of this to debug sometimes and with bad eyes hopefully this will be better

peoplearestrange committed:
Removed hide most verb

Removed the remaining code for "hide most" verb as its now not needed after the reshuffle

peoplearestrange committed:
Changes category

Set admin notice = Server

Cyberboss commented:
God given PR

peoplearestrange reviewed (commented):
Made the changes

PKPenguin321 commented:
PSA to admins, if you use tabs instead of the command line you're doing it wrong

peoplearestrange commented:
> PSA to admins, if you use tabs instead of the command line you're doing it wrong

True but its a helluva lot easier to teach people buttons when learning. Plus not everything is labelled consistently so remembering the right verb isn't always that simple

PKPenguin321 commented:
>remembering the right verb isn't always that simple

learn to use the autofill to search for verbs faster, trust me it's super worth it

peoplearestrange commented:
Ergggg, because of the way it sorts things Administration is now last in the tabs. Like this:
![image](https://user-images.githubusercontent.com/7211040/73782713-77408f00-478a-11ea-9b79-9dfe5b51739b.png)

I feel like, this might be an issue. Will probably make "Administration" back into "Admin" to avoid this.

peoplearestrange committed:
Changes to Category

changes Administration to Admin

peoplearestrange committed:
Changes to category

changes Administration to Admin

peoplearestrange committed:
Changes to Category

changes Administration to Admin

peoplearestrange committed:
changes to category

Changes Administration to Admin

peoplearestrange committed:
changes to category

changes Administration to Admin

peoplearestrange committed:
changes to category

changes Administration to Admin

peoplearestrange committed:
changes to category

Changes Administration to Admin

peoplearestrange committed:
changes to category

changes Administration to Admin

peoplearestrange committed:
changes to category

Changes Administration to Admin

peoplearestrange committed:
Changes to category

changes Administration to Admin

peoplearestrange committed:
changes to category

changes Administration to Admin

peoplearestrange committed:
Changes to category

changes Administration to Admin

peoplearestrange committed:
changes to category

changes Administration to Admin

peoplearestrange committed:
changes to category

changes Administration to Admin

peoplearestrange commented:
Changes made, I think thats it. Will download and test it

itseasytosee commented:
Why is start game not in server and where is it now.

cacogen commented:
Start Now is under Admin - Events. Should be in Server with things like End
Round, Reboot World, Change Map, etc.(

On Thu, 6 Feb 2020 at 1:04 PM, itseasytosee <notifications@github.com>
wrote:

> Why is start game not in server and where is it now.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tgstation/tgstation/pull/49184?email_source=notifications&email_token=AF7NO6QUHVHI64OGOHV3CLLRBNHX3A5CNFSM4KPQ6RKKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEK5N7IA#issuecomment-582672288>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF7NO6RFKL4N24FG355V2MTRBNHX3ANCNFSM4KPQ6RKA>
> .
>

peoplearestrange commented:
Its in events because thats when its usually used. When its paused for an event and then resumed.
If it annoys people I could move it to server

Cyberboss commented:
#49216
